src/Sylius/Bundle/ApiBundle/Controller/UpdateProductTaxonPositionController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ApiBundle\Controller;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sylius\Component\Core\Model\ProductTaxonInterface;
+use Sylius\Component\Core\Repository\ProductTaxonRepositoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+final class UpdateProductTaxonPositionController
+{
+    /**
+     * @var ProductTaxonRepositoryInterface
+     */
+    private $productTaxonRepository;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $manager;
+
+    /**
+     * @param RepositoryInterface $productTaxonRepository
+     * @param EntityManagerInterface $manager
+     */
+    public function __construct(
+        RepositoryInterface $productTaxonRepository,
+        EntityManagerInterface $manager
+    ) {
+        $this->productTaxonRepository = $productTaxonRepository;
+        $this->manager = $manager;
+    }
+
+    /**
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function updatePositionsAction(Request $request)
+    {
+        $positionsData = $request->request->get('positionsData');
+
+        if (in_array($request->getMethod(), ['POST', 'PUT', 'PATCH'], true) && null !== $positionsData) {
```

Comment:
What would you say for
```php
if (!in_array($request->getMethod(), ['POST', 'PUT', 'PATCH'], true) || null === $positionsData) {
    return new JsonResponse();
}
```
? It would be one less indentation level.